.pre-commit-config.yaml

```diff
@@ -59,3 +59,18 @@ repos:
           - -w      # autofix in place
           - -d      # show diff
           - -i=2    # indent 2 spaces
+
+  # Example usage of dotnet-format: pre-commit run dotnet-format --all-files
+  - repo: local
```

Comment:
If we self-include by specifying this repo here at `master`, we can avoid this duplication and possible drift between config and hook-def.